test/Microsoft.ML.Core.Tests/UnitTests/DvTypes.cs

```diff
@@ -10,60 +10,6 @@ namespace Microsoft.ML.Runtime.RunTests
 {
     public sealed class DvTypeTests
     {
-        [Fact]
-        public void TestComparableDvInt4()
```

Comment:
>TestComparableDvInt4 [](start = 20, length = 20)

why you removing whole test? Is DvInt4 gone already or in this PR?

Reply:
There is another PR that removes DvInt*

Reply:
Its gone in DvInt4 PR.

---
In reply to: [211109511](https://github.com/dotnet/machinelearning/pull/692#discussion_r211109511) [](ancestors = 211109511)